doc/Makefile

```diff
@@ -1,240 +1,20 @@
-# Makefile for Sphinx documentation
+# Minimal makefile for Sphinx documentation
 #
```

Comment:
Generating the docs produces some errors like:
`AttributeError: module 'fparser.two.symbol_table' has no attribute 'ModuleUse'`

And many warnings, no need to address all but there are some marked as CRITICAL.. e.g.:

```
src/fparser/common/readfortran.py:docstring of fparser.common.readfortran.Line:4: CRITICAL: Unexpected section title.
src/fparser/common/readfortran.py:docstring of fparser.common.readfortran.MultiLine:7: CRITICAL: Unexpected section title.
src/fparser/common/utils.py:docstring of fparser.common.utils:3: CRITICAL: Missing matching underline for section title overline.
src/fparser/one/statements.py:docstring of fparser.one.statements.Comment:2: CRITICAL: Unexpected section title.
src/fparser/one/statements.py:docstring of fparser.one.statements.Comment:2: CRITICAL: Unexpected section title.
src/fparser/two/pattern_tools.py:docstring of fparser.two.pattern_tools:3: CRITICAL: Missing matching underline for section title overline.
```
can you check if these are easy to solve?

Reply:
I can't find any 'CRITICAL' now so hopefully that's done.